components/List/List.js

```diff
@@ -0,0 +1,39 @@
+import {
+  StyledImage,
+  StyledList,
+  StyledCard,
+  StyledName,
+  StyledCuisine,
+  StyledRating,
+  StyledPicture,
+} from "./List.styles";
+
+//when data bank is create, id should be changed to _id
+export default function Card({ restaurants }) {
+  return (
+    <StyledList>
+      {restaurants.map((restaurant) => (
+        <StyledCard key={restaurant.id}>
```

Comment:
The name does not reflect that it's a `li`, this might cause confusion later on.